# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libnids/gcc44.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/killtcp.c.orig	2006-05-08 22:05:51.000000000 +0200
+++ ./src/killtcp.c	2009-07-02 17:42:29.000000000 +0200
@@ -118,6 +118,6 @@
     if (initialized)
 	abort();
 }
-#elif
+#else
 #error Something wrong with LIBNET_VER
 #endif

